app_dart/lib/src/service/luci.dart

```diff
@@ -165,11 +164,8 @@ class LuciService {
   /// predefined in cocoon config.
   ///
   /// Latest builds of each builder will be returned from new to old.
-  Future<Iterable<Build>> getBuilds(String? repo, bool requireTaskName, List<LuciBuilder> builders) async {
+  Future<Iterable<Build>> getBuilds(bool requireTaskName, List<LuciBuilder> builders) async {
     bool includeBuilder(LuciBuilder builder) {
-      if (repo != null && builder.repo != repo) {
```

Comment:
This logic isn't necessary. Flutter's LUCI project has all builders globally available to every repo.

Thus, builders are repos specific because of the ci.yaml, and the passed `builders` here has already filtered it.